### PR TITLE
use Lomap package's bindings

### PR DESCRIPTION
**Pull request by richardjgowers**

### CODE REVIEW

File: openfe/setup/atom_mapping/lomap_scorers.py
Comment (dwhswenson) [inline openfe/setup/atom_mapping/lomap_scorers.py:3]:
```suggestion
from lomap.gufe_bindings.scorers import default_lomap_score  # hack for mypy
from lomap.gufe_bindings.scorers import *
```

File: openfe/setup/atom_mapping/lomap_scorers.py
Comment (dwhswenson) [inline openfe/setup/atom_mapping/lomap_scorers.py:5]:
```suggestion
```

Now it got even weirder....

File: docs/environment.yaml
Comment (mikemhenry) [inline docs/environment.yaml:23]: Is it time for another lomap release?

Comment (dwhswenson) [reply docs/environment.yaml:23]: There are concerns about backward compatibility breaks for current users. We need to give them (namely, Mary) a migration window.

**Review by dwhswenson — APPROVED** [review]
Okay, I'm happy with this now. Got it to pass mypy and RTD.

**WARNING!** This is going to ruin everyone's dev environments (unless you happen to already have LOMAP at main).

File: docs/environment.yaml
Comment (IAlibay) [reply docs/environment.yaml:23]: if we can't do a release right now then we can't merge this if want to have a release by end of week.

**Review by IAlibay — CHANGES_REQUESTED** [review]
blocking for now

File: docs/environment.yaml
Comment (dwhswenson) [reply docs/environment.yaml:23]: Fine by me. We need to get a migration writeup in place, IMO. I'll remove this from the 0.12.0 milestone.

Comment (mikemhenry) [reply docs/environment.yaml:23]: Fine with moving this to 0.12 but I don't understand:

> There are concerns about backward compatibility breaks for current users. We need to give them (namely, Mary) a migration window.

'lomap2 < $NEW_VERSION'

Isn't enough

**Review by IAlibay — APPROVED** [review]
I'm eagerly approving this so I'm not blocking - the new release of Lomap is in progress, so this should be good to go.

File: docs/reference/api/ligand_network.rst
Comment (richardjgowers) [inline docs/reference/api/ligand_network.rst:93]: this is all causing warnings for now, so I'll remove and raise an issue about adding this documentation back

**Pull request merged**
